src/Files.App/Utils/Cloud/CloudProviders.cs

```diff
@@ -45,6 +45,8 @@ public enum CloudProviders
 
 		LucidLink,
 
-		kDrive
+		kDrive,
+
+		Sync
```

Comment:
I think we should prefix this provider with the publisher's name to avoid confusion.

Reply:
What do you mean?

Reply:
They company also called sync. So unless you want syncsync I don't think it would work

Reply:
How about `syncCloud`?

Reply:
The Enum name is already CloudProviders

Reply:
I think the company is called sync.com, so `syncCom` might also be an alternative

Reply:
That works, the main thing is to make clear that this is a provider and not a generic fallback.